lib/ae_mdw_web/websocket/broadcaster.ex

```diff
@@ -4,19 +4,23 @@ defmodule AeMdwWeb.Websocket.Broadcaster do
   """
   use GenServer
 
+  alias AeMdw.EtsCache
+
   require Ex2ms
 
   @dialyzer {:no_return, broadcast: 2}
 
-  @no_state %{}
+  @hashes_table :broadcast_hashes
+  @expiration_minutes 120
   @subs_target_channels :subs_target_channels
 
   @spec start_link(any()) :: GenServer.on_start()
   def start_link(_arg), do: GenServer.start_link(__MODULE__, :ok, name: __MODULE__)
 
   @impl GenServer
   def init(:ok) do
-    {:ok, @no_state}
+    EtsCache.new(@hashes_table, @expiration_minutes)
```

Comment:
One last thing, I would move the creation of this named ETS to outside the GenServer init in case it restarts.

Reply:
Done